controls/nist_ocp4.yml

```diff
@@ -15888,17 +15888,40 @@ controls:
   - high
   - moderate
 - id: SI-7(2)
-  status: pending
+  status: automated
   notes: |-
+    The OpenShift Platform provides the File Integrity Operator to do
+    integrity verification of files and information on the nodes. [1]
+
+    The operator can be configured to perform integrity checks at user-defined
+    intervals. [2]
+
+    A Prometheus rule will be added for the File Integrity Operator to alert
+    organization defined personnel for any integrity check failure. 
```

Comment:
Didn't we already add the rule?

Reply:
We didn't add those rules into SI-7(2) controls